presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloSplitManager.java

```diff
@@ -57,7 +57,12 @@ public AccumuloSplitManager(
     }
 
     @Override
-    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorTableLayoutHandle layout, SplitSchedulingStrategy splitSchedulingStrategy)
+    public ConnectorSplitSource getSplits(
+            ConnectorTransactionHandle transactionHandle,
+            ConnectorSession session,
+            ConnectorTableLayoutHandle layout,
+            SplitSchedulingStrategy splitSchedulingStrategy,
+            boolean schedulerUsesAddresses)
```

Comment:
@arhimondr : This would assume we will not have other `NodeSelector`  (although it's likely the case). So I think it should be something more high-level about the intention -- the message to connector is "hey the host address you returned will not be ignored". But maybe we can come up with some better variable name :)

Reply:
`preferSplitHostAddresses`?  Just a random thought and maybe not good. :)